internal/controller/mantlerestore_controller_test.go

```diff
@@ -212,6 +212,52 @@ func (test *mantleRestoreControllerUnitTest) testCreateRestoringPVC() {
 		restore = test.restoreResource()
 	})
 
+	It("should return an error if the PVC already exists and has no RestoredBy annotation", func(ctx SpecContext) {
+		err := k8sClient.Create(ctx, &corev1.PersistentVolumeClaim{
+			ObjectMeta: metav1.ObjectMeta{Name: restore.Name, Namespace: test.tenantNamespace},
+			Spec: corev1.PersistentVolumeClaimSpec{
+				AccessModes: []corev1.PersistentVolumeAccessMode{corev1.ReadWriteOnce},
+				Resources: corev1.VolumeResourceRequirements{
+					Requests: test.srcPVC.Spec.Resources.Requests,
+				},
+				StorageClassName: test.srcPVC.Spec.StorageClassName,
+
+				// We intentionally leave volumeName unset here. Setting it
+				// would cause createOrUpdate to fail since volumeName is
+				// immutable. Our goal is to test that createOrUpdate isn't
+				// applied to this PVC, so we keep it empty.
+			},
+		})
+		Expect(err).NotTo(HaveOccurred())
+
+		var pvc0 corev1.PersistentVolumeClaim
```

Comment:
```suggestion
		var pvc0 corev1.PersistentVolumeClaim
```
Since `pvc0` and `pvc1` mean the same K8s object, how about renaming these variables to `pvc` and `updatedPVC`?